### PR TITLE
Add updated production deployment guidelines

### PR DESCRIPTION
**Pull request by wgordon17**

### CODE REVIEW

File: contributing_to_docs/docs_production_deployment.adoc [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+using a https://github.com/openshift/source-to-image[source-to-image] build pipeline.
+
+The source-to-image builder image is built from a https://github.com/openshift-cs/docs-builder/[community project in GitHub]
+and published to https://quay.io/repository/openshift-cs/docs-builder.
+
+== Documentation deployment
+Deploying the OpenShift documentation is simplified by using a
+https://github.com/openshift-cs/docs-builder/blob/main/template.yaml[pre-built OpenShift template YAML].
+
+You can use the following command to deploy the OpenShift Container Platform (commercial) documentation
```
Comment (mburke5678) [inline contributing_to_docs/docs_production_deployment.adoc:23]: s/documentation/documentation:

Comment (wgordon17) [reply contributing_to_docs/docs_production_deployment.adoc:23]: Ha! I thought this was a spelling error...took way to much time to realize you wanted a `:` at the end 😉

Comment (wgordon17) [inline contributing_to_docs/docs_production_deployment.adoc:23]:
```suggestion
You can use the following command to deploy the OpenShift Container Platform (commercial) documentation:
```

**Comment by mburke5678** [comment]
@wgordon17 LGTM. Do we need to add anything about who would want to use these commands and why? The original doc has _You can deploy to your own OpenShift cluster for development._. (I, personally, have no idea what that means!)

**Comment by wgordon17** [comment]
@mburke5678 Thanks for the review! I'm going to vote _against_ including the "why"...just kind of putting it out there as _general_ information ¯\_(ツ)_/¯ 

@vikram-redhat, if (when you link to this from elsewhere) you wanted to include something brief, be my guest 😊

**Comment by wgordon17** [comment]
@vikram-redhat This PR should be good to go now btw 👍

**Pull request merged**